web-marketplace/src/apollo.tsx

```diff
@@ -54,6 +67,42 @@ export const AuthApolloProvider = ({
     },
   };
 
+  const errorLink = onError(({ operation, networkError, forward }) => {
+    if (
+      networkError &&
+      networkError.message.includes('status code 403') &&
+      operation.operationName !== 'GetCurrentAccount'
```

Comment:
We should not intercept this request (for now) as it is currently done outside of `react-query` and since it is executed several times on each page it can unnecessary requests replaying.